lib/status/status.go

```diff
@@ -522,15 +522,6 @@ func fromPlanetAgent(ctx context.Context, local bool, servers []storage.Server)
 		nodes = fromSystemStatus(*status)
 	}
 
-	// Collected system status may exclude the status of some nodes due to
-	// network partition. If offline nodes are detected, degrade system status.
-	for _, node := range nodes {
```

Comment:
Is this change intended?

Reply:
Yeah. While testing shrink op I realized that this added check causes the system status to fall into a degraded state temporarily during the shrink operation.

Since we are adding the serf reconciler, I figured it would be better to have the system status show healthy cluster status temporarily if serf is out of sync than to have the system status fall into a degraded state during shrink ops.